### PR TITLE
Ui improvements

### PR DESCRIPTION
**Pull request by skkr**
This PR includes several UI fixes and improvements

- Updated HIW diagram with more readable text
- Resolved issue with homepage content overlflow
- Corrected header position (overlaping the hero)
- Corrected header button animation
- Corrected several design details in the index page
- Corrected size, position and styles of video modal
- Corrected secondary hero paddings and margins of child elements
- Updated some cads styles to have a different design in mobile (hero cards)

Closes https://github.com/windingtree/homepage/issues/177

### CODE REVIEW

File: src/features/Home/InThePress/Quotes/data.js [hunk 1]
```diff
@@ -23,6 +23,8 @@ const quotes: QuoteType[] = [
     text: 'We are proud to be one of the first airline groups to join Winding Tree to develop Blockchain technology” said Sonia Barrière, EVP Strategy and Innovation at Air France-KLM.  “With a long-standing commitment to innovation with start-ups and partners, Air France-KLM is constantly creating the future of travel and devising solutions to make the travel experience easier and more personalized. With Blockchain technology, we aim to revolutionize exchanges within the travel industry for our customers, companies and start-ups...',
     href: 'https://www.airfranceklm.com/en/news/air-france-klm-partners-winding-tree-strengthen-innovation-travel-industry-using-blockchain',
     imgSrc: airFrance,
+    imgWidth: 180,
```
Comment (MatiasOS) [inline src/features/Home/InThePress/Quotes/data.js]: Please, can you add it to QuoteType between `imgSrc` and `user?` ?
```
imgWidth: number,
imgTopPosition: number,
```

File: src/features/Main/index.jsx [hunk 2]
```diff
@@ -10,62 +10,81 @@ type PropsType = {
...
+class WTNavbar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+        expanded: false,
+    };
+  }
+
+  toggle() {
+    console.log('6654');
```
Comment (MatiasOS) [inline src/features/Main/index.jsx]: You forgot to remove this line

File: src/features/Main/index.jsx [hunk 3]
```diff
@@ -10,62 +10,81 @@ type PropsType = {
...
+    this.setState({expanded: !this.state.expanded});
+  }
+
+  render() {
+    let self = this;
```
Comment (MatiasOS) [inline src/features/Main/index.jsx]: Please, change this line to ` const { expanded } = this.state;`

File: src/features/Main/index.jsx [hunk 4]
```diff
@@ -10,62 +10,81 @@ type PropsType = {
...
+      return (
+        <header id="app-header" className="header-sticky">
+          <Navbar expand="lg" variant="animated" className={`navbar-nav--animated-btn ${self.state.expanded ? 'is-open' : null}`}>
```
Comment (MatiasOS) [inline src/features/Main/index.jsx]: And here, you can use directly `....   ${expanded ? ... }`

**Comment by skkr** [comment]
Had to force push, since I made changes to the last commit after a first push. Anyway, all the previous commits where reviewed already.

**Comment by skkr** [comment]
> 
> 
> Community page still have scrolling bug, because of contributors slick

Done!

**Pull request merged**